src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -26,7 +26,7 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.IO.CSV.WriteToFile</oldName>
-    <newName>DSCore.ImportExport.ExportCSV</newName>
+    <newName>DSCore.ImportExport.Import.ExportCSV</newName>
```

Comment:
Though we are working on different nodes, but we are both migrating nodes to `Import\Export` category, so there would be some overlap between our tasks.